precompile/testutils/test_predicate.go

```diff
@@ -0,0 +1,87 @@
+// (c) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package testutils
+
+import (
+	"testing"
+	"time"
+
+	"github.com/ava-labs/subnet-evm/precompile/precompileconfig"
+	"github.com/stretchr/testify/require"
+)
+
+type PredicateTest struct {
```

Comment:
could we add some comment about how this struct is intended for use?

Reply:
Yup